share_extension/android/extension_channels/extension_channels.js

```diff
@@ -238,7 +309,7 @@ export default class ExtensionTeam extends PureComponent {
 const getStyleSheet = makeStyleSheetFromTheme((theme) => {
     return {
         flex: {
-            flex: 1,
+            flexGrow: 0,
```

Comment:
Won't `flex: 1` fill the allocated space, effectively doing the same "no resize" action as `flexGrow: 0` above?

My guidance is based on the footnote in the official spec of `flexGrow` (see [here](https://drafts.csswg.org/css-flexbox/#flex-grow-property)) which favours `flex: n` over `flexGrow`. But perhaps there is a legitimate use case/side-effect being handled in your case here which I'd like to understand better?

Reply:
Yes, `flex: 1` and `flexGrow: 0` indeed are the same when displaying results.

However, when there are “No Results”, `flex: 1` causes the SectionList/”No Results” to fill up the screen and thus push down the activity spinner below it. In this case, I just wanted the spinner to be viewable right below the “No Results” text, which `flexGrow: 0` does here.

Reply:
Aha... the good ol' "No Results" case. 👍